app.js

```diff
@@ -0,0 +1,69 @@
+// import function(s)
+import { getRandomThrow, checkResults } from './random-throw.js';
+
+//DOM elements
+const myButton = document.getElementById('guess-button');
+const computerSelection = document.getElementById('cpu-pick');
```

Comment:
You probably want to name this variable something like `computerSelectionDisplay` as you are calling a different variable `computerSelection` on line 37